receptorctl/tests/conftest.py

```diff
@@ -389,3 +390,23 @@ def f_invoke_as_json(command, args: list = []):
         return result, json_output
 
     return f_invoke_as_json
+
+
+def pytest_configure(config):
+    config.addinivalue_line(
+        "markers",
+        "github(): this mark skips the tests for the given open GitHub issues",
+    )
+
+
+def pytest_runtest_setup(item):
+    gh_markers = [mark.args[0] for mark in item.iter_markers(name="github")]
+    if gh_markers:
+        issue_url = gh_markers[0]
+        with urllib.request.urlopen(issue_url) as f:
```

Comment:
Yes. You will need to specify the marker like this - 

```python
import pytest

@pytest.mark.github('https://api.github.com/repos/ansible/receptor/issues/441')
def test_c():
    print("This is Test C")
```

I choose this routine in order to avoid additional dependency on third-party library.

Reply:
we can simplify the logic further to specify only the issue number or pull request number. But for starters, I think this OK.

Reply:
I think this is pretty difficult to use. Normally you'd just visit an issue in your browser. We should be able to copy / paste that URL instead of having to construct this API endpoint. We use https://github.com/jlaska/pytest-github in other projects, so we're not maintaining anything extra by also using it here.